swagger.yaml

```diff
@@ -1,75 +1,298 @@
 swagger: '2.0'
+
 info:
-  title: Web API Skeleton
-  description: Skeleton for Dropwizard Web APIs
-  version: '0.0'
-  license:
-    name: GNU Affero General Public License Version 3
-    url: http://www.gnu.org/licenses/agpl-3.0.en.html
-externalDocs:
-  description: GitHub Repository
-  url: https://github.com/osu-mist/web-api-skeleton
-host: localhost
+  description: "This provides an API for interacting with a database of cards from the video game Slay the Spire. It allows users to retrieve cards, add new cards, update cards, and delete cards from the database."
+  version: "1.0"
+  title: "Slay the Spire Cards API"
+
+tags:
+- name: "cards"
+  description: "Contains information on cards"
+
 schemes:
-  - http
-basePath: /api/v0
-consumes:
-  - text/plain
-produces:
-  - text/plain
+- "https"
+
 paths:
-  /:
+  /cards:
+
     get:
-      summary: Get information
-      description: Build and runtime information about the application.
+      tags:
+      - "cards"
+      summary: "Retrieve cards based on search parameters"
+      description: ""
```

Comment:
Looks like you aren't using the description fields for any of the endpoints. Could you remove them instead?